devtools/test_dashboard/buttons.js

```diff
@@ -11,7 +13,7 @@ anchor.style.width = '1000px';
 
 function plotButtons(plots, figDir) {
 
-    Object.keys(plots).forEach( function (plotname) {
+    Object.keys(plots).forEach(function (plotname) {
```

Comment:
do we want the space between `function` and `(plotname)`
(sorry I think I'm a little confused with the controversial spacing rule)